sample_consensus/include/pcl/sample_consensus/impl/sac_model_cone.hpp

```diff
@@ -344,14 +343,20 @@ pcl::SampleConsensusModelCone<PointT, PointNT>::optimizeModelCoefficients (
     return;
   }
 
-  OptimizationFunctor functor (this, inliers);
```

Comment:
Should the old OptimizationFunctors be removed? I guess they are not used now?

Reply:
You are right, I removed them